src/controllers/api/Transaction.ts

```diff
@@ -55,7 +55,16 @@ export default {
   },
   find: async (req: Request, res: Response) => {
     const where = req.query.where ? JSON.parse(req.query.where) : {};
-    const result = await Transactions.findAll({ where });
+    const limit = req.query.limit ? Number(req.query.limit) : 1;
```

Comment:
limit, offset, sortBy, sort should be defined in the constant file
for example: 
import { QUERY_LIMIT } from '../constant';
const limit = req.query.limit ? Number(req.query.limit) : QUERY_LIMIT;

sort should be default 'created_at';